### DISCUSSION TITLE
Using excalidraw.com for commercial purposes

### DISCUSSION THREAD

**Discussion by mkasprz** [excalidraw/excalidraw#6665] [Q&A]
Hi,

Can I legally use excalidraw.com for commercial purposes, while collaborating with others, like while giving someone a lesson or creating some mock-ups for a commercial product?

I've been looking for something like Terms of Service document to answer this question, but the only think I've found for now is [Excalidraw+ TERMS OF USE](https://plus.excalidraw.com/terms-of-service) and I'm not sure if that's relevant (I've [asked about it](https://github.com/excalidraw/excalidraw/discussions/6664)).

Assuming it is though the following paragraph concerns me a bit:

> Provided that you are eligible to use the Site, you are granted a limited license to access and use the Site and to download or print a copy of any portion of the Content to which you have properly gained access solely for your personal, non-commercial use. We reserve all rights not expressly granted to you in and to the Site, the Content and the Marks.

The next one says that I do can distribute drawings creating in the editor commercially, but I wonder if the one above doesn't prohibit just **using** the editor with someone to create such drawings.

Thank You in advance.

**Comment by microSoftware** [comment]
Also interested to know if we can self-host excalidraw and integrate it into a commercial product? 
If yes, should we mention "excalidraw" ?

**Reply by mkasprz** [reply]
Excalidraw is distributed under MIT license which, I'm pretty sure of, allows integrating it into commercial product. It seems there is [only one thing required](https://github.com/excalidraw/excalidraw/blob/master/LICENSE) - this time at least:

> The above copyright notice and this permission notice shall be included in all
> copies or substantial portions of the Software.

That, I think, means, that You should include that notice when distributing Your product (which doesn't mean Your whole product have to be distributed under this license as far as I know) - for example in some ".txt" file or something. If by it You mean "mentioning" "excalidraw" then, it seems, yes. If You meant just having something like "powered by Excalidraw", then. I think, no, it's not required.

And just to clarify - my question was about the excalidraw.com - that is the instance of Excalidraw, not the source code.

**Comment by joaofelipelliccione** [comment]
After carefully reviewing Excalidraw's documentation (refer to the attached photo), I am confident that it is an excellent choice for integration into commercial products.

Moreover, it would be counterintuitive for Excalidraw to consistently provide updates to their npm package if they did not intend for users to create new applications.

From my perspective, cultivating a diverse global user and developer base is tremendously advantageous for an open-source project. This allows users to contribute by reporting bugs and suggesting improvements.

Encouraging other entrepreneurs to develop applications powered by Excalidraw contributes to the project's growth and expansion.

In my assessment, Excalidraw demonstrates a strong commitment to the common good, prioritizing ideals over financial considerations and competition. It embodies the spirit of idealism by creating an exceptional tool for universal adoption.

It is important to acknowledge and credit Excalidraw on your website or app. Consider including a statement like "Powered by Excalidraw" along with a link to its GitHub repository or documentation as a meaningful gesture of appreciation.

Additionally, if your product becomes financially successful, consider making a donation to the Excalidraw project or supporting it in some other way. This would be a great way to give back and show gratitude.

These are my thoughts on the matter!

![Screenshot_20230713_081859_Chrome.jpg](https://github.com/excalidraw/excalidraw/assets/83513721/3356a9b3-fcc1-4bae-9884-429bd70cc5c2)

**Reply by microSoftware** [reply]
Okay thanks

**Comment by joaofelipelliccione** [comment]
@microSoftware and @mkasprz,

I forgot to attach another image that provides even clearer information than the previous one.
![Captura de Tela 2023-07-13 às 11 55 31](https://github.com/excalidraw/excalidraw/assets/83513721/665f356a-beb4-4c2f-a2b5-34ecd10bcdd6)

**Reply by mkasprz** [reply]
@joaofelipelliccione , hi, thank You for the comments and all.

To be honest it does not answer my question. Integrating Excalidraw into another project and using excalidraw.com are two different things I believe, as excalidraw.com may have a totally different Terms of Use. It's like comparing source code terms of use to the application instance terms of use and I'm asking about the latter.

**Comment by ad1992** [comment]
@mkasprz with the above detailed responses I am hoping the query is resolved!
You can definitely integrate Excalidraw into your own app, thats why we the build the npm package to make integration easier :)

**Reply by mkasprz** [reply]
@ad1992 , hi, thank You for the comment.

No, it's not. My question is not about integrating Excalidraw into my own app. My question is about using excalidraw.com instance for commercial purposes. That's two different things.

I know I can use Excalidraw source code or NPM package to integrate it into my own application as I understand it's distributed under MIT license, but the excalidraw.com instance may have totally different Terms of Use and may not allow commercial usage - that's why I'm asking about it.

**Comment by Cethrivias** [comment]
Hey. I was interested in the same thing and it seems that their model is the same as notion. Provide the base product completely for free for everyone. (including commercial use) and then charge for extra features. So I guess your use (giving presentations at work) is totally safe.
In the free tier you can see "oss software for devs"
![image](https://github.com/user-attachments/assets/6c3fa4e2-d3d6-47db-8577-07ab98b4c623)
